pathd/path_pcep.h

```diff
@@ -84,6 +85,9 @@ struct pce_opts {
 	struct ipaddr addr;
 	short port;
 	bool draft07;
+	uint8_t precedence;
+	bool is_best;
```

Comment:
This doesn't look like it belong here, these are not options.

Reply:
The precedence is a config option and the is_best and previous_best are 'calculated' but if not a new struct must be created and mantained so for now this is a good place